swagger.yaml

```diff
@@ -0,0 +1,59 @@
+swagger: '2.0'
+info:
+  title: Odin-ah-backend
+  description: " A Social platform for the creative at heart."
+  version: v1
+host: 127.0.0.1:3001
+produces:
+- application/json
+schemes:
+- https
```

Comment:
change schema to `http`

Reply:
We can have both

Reply:
If we can have both, that would be great